lib/src/request/request_cubit_builder.dart

```diff
@@ -66,9 +66,12 @@ class RequestCubitBuilder<TOut, TError> extends StatelessWidget {
           RequestRefreshState(:final data) => data != null
               ? builder(context, data)
               : onLoading?.call(context) ?? config.onLoading(context),
-          RequestErrorState() =>
-            onError?.call(context, state, onErrorCallback) ??
-                config.onError(context, state, onErrorCallback),
+          RequestErrorState() => onError?.call(
+                context,
+                state,
+                onErrorCallback ?? cubit.refresh,
```

Comment:
https://github.com/leancodepl/leancode_cubit_utils/blob/649ead738a312c7cb9626cd13446ec3af36147d8/lib/src/paginated/paginated_cubit_layout.dart#L167

We use run() there - I think it should be consistent.

Reply:
in BaseRequestCubit only refresh is available